diff --git a/SkyMPC.pro b/SkyMPC.pro
index c54f130..8715843 100644
--- a/SkyMPC.pro
+++ b/SkyMPC.pro
@@ -11,8 +11,6 @@ TEMPLATE = app
 
 CONFIG += c++11
 
-DESTDIR = $$PWD/_bin
-
 unix:QMAKE_CXXFLAGS += -Wall -Wextra -Werror=return-type -Werror=trigraphs -Wno-switch
 unix:QMAKE_RPATHDIR += $ORIGIN
 
@@ -75,7 +73,7 @@ SOURCES += src/main.cpp \
 	src/TinyConnectionDialog.cpp \
 	src/BasicConnectionDialog.cpp \
 	src/AskRemoveOverlappedFileDialog.cpp \
-    version.c \
+#    version.c \
     src/StatusLabel.cpp \
     src/SettingsDialog.cpp \
     src/AbstractSettingForm.cpp \
@@ -157,6 +155,9 @@ FORMS    += src/MainWindow.ui \
 RESOURCES += \
 	resources.qrc
 
+target.path = /opt
+INSTALLS += target
+
 win32 {
 	LIBS += user32.lib
 	LIBS += shell32.lib
diff --git a/src/AboutDialog.cpp b/src/AboutDialog.cpp
index 90c7435..ab3ca53 100644
--- a/src/AboutDialog.cpp
+++ b/src/AboutDialog.cpp
@@ -21,8 +21,8 @@ AboutDialog::AboutDialog(QWidget *parent) :
 
 	setWindowTitle(tr("About SkyMPC"));
 
-	ui->label_title->setText(QString("SkyMPC, v%1 (%2)").arg(product_version).arg(source_revision));
-	ui->label_copyright->setText(QString("Copyright (C) %1 S.Fuchita").arg(copyright_year));
+//	ui->label_title->setText(QString("SkyMPC, v%1 (%2)").arg(product_version).arg(source_revision));
+//	ui->label_copyright->setText(QString("Copyright (C) %1 S.Fuchita").arg(copyright_year));
 	ui->label_twitter->setText("(@soramimi_jp)");
 	QString t = QString("Qt %1").arg(qVersion());
 #if defined(_MSC_VER)
